etc/encoding/adaptive-streaming-movies.properties

```diff
@@ -93,3 +93,45 @@ profile.adaptive-parallel.http.ffmpeg.command = -i #{in.video.path} \
   -c:v libx264 -crf 23 -maxrate 800k -bufsize 800k -profile:v high -level 4.0 -pix_fmt yuv420p \
     -x264opts keyint=25:min-keyint=25:no-scenecut -movflags +faststart -vf scale='-2:min(ih\\,480)',fps=fps=25 \
     -c:a aac -b:a 32k -ac 1 -f mp4 #{out.dir}/#{out.name}#{out.suffix.480p-quality}
+
+profile.studio.adaptive-parallel.http.name = parallel encoding of studio video in multiple qualities for adaptive streaming
+profile.studio.adaptive-parallel.http.input = visual
+profile.studio.adaptive-parallel.http.output = visual
+profile.studio.adaptive-parallel.http.suffix.2160p-quality = -2160p-quality.mp4
+profile.studio.adaptive-parallel.http.suffix.1440p-quality = -1440p-quality.mp4
+profile.studio.adaptive-parallel.http.suffix.1080p-quality = -1080p-quality.mp4
+profile.studio.adaptive-parallel.http.suffix.720p-quality = -720p-quality.mp4
+profile.studio.adaptive-parallel.http.suffix.360p-quality = -360p-quality.mp4
+profile.studio.adaptive-parallel.http.jobload = 4
+profile.studio.adaptive-parallel.http.ffmpeg.command.if-height-geq-2160 = \
+  -filter:v scale=w=3840:h='trunc(3840/dar/2)*2':force_original_aspect_ratio=disable,setsar=1/1,fps=fps=25 \
+    -c:v libx264 -preset slower -tune film -pix_fmt yuv420p \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -crf 20 -maxrate 14800k -bufsize 18000k \
+    -c:a aac -b:a 196k -ac 2 \
+    -movflags +faststart #{out.dir}/#{out.name}#{out.suffix.2160p-quality}
+profile.studio.adaptive-parallel.http.ffmpeg.command.if-height-geq-1440 = \
+  -filter:v scale=w=2560:h='trunc(2560/dar/2)*2':force_original_aspect_ratio=disable,setsar=1/1,fps=fps=25 \
+    -c:v libx264 -preset slower -tune film -pix_fmt yuv420p \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -crf 20 -maxrate 9800k -bufsize 14000k \
+    -c:a aac -b:a 128k -ac 2 \
+    -movflags +faststart #{out.dir}/#{out.name}#{out.suffix.1440p-quality}
+profile.studio.adaptive-parallel.http.ffmpeg.command.if-height-geq-1080 = \
+  -filter:v scale=w=1920:h='trunc(1920/dar/2)*2':force_original_aspect_ratio=disable,setsar=1/1,fps=fps=25 \
+    -c:v libx264 -preset slower -tune film -pix_fmt yuv420p \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -crf 20 -maxrate 4800k -bufsize 8000k \
+    -c:a aac -b:a 128k -ac 2 \
+    -movflags +faststart #{out.dir}/#{out.name}#{out.suffix.1080p-quality}
+profile.studio.adaptive-parallel.http.ffmpeg.command = -i #{in.video.path} \
+  #{if-height-geq-2160} \
+  #{if-height-geq-1440} \
+  #{if-height-geq-1080} \
+  -filter:v scale=w=1280:h='trunc(1280/dar/2)*2':force_original_aspect_ratio=disable,setsar=1/1,fps=fps=25 \
```

Comment:
This is identical to the simpler

```suggestion
  -filter:v scale=w=1280:h='trunc(1280/dar/2)*2',setsar=1,fps=25 \
```

…though I'm wondering if it's better to use…

```
-filter:v scale=w=1280:720:force_original_aspect_ratio=decrease:force_divisible_by=2,fps=25 \
```
…which ensures that the video size does not exceed `1280:720`?
The same goes for the other filters.

Reply:
You included an `fps` filter in the pre-processing anyway, so there is no need to include it here again, is there?

Reply:
The latest ffmpeg build from rpmfusion on fedora do not know the paramater `force_divisible_by` (anymore?!)

```
[Parsed_scale_0 @ 0x55c95e30efc0] Option 'force_divisible_by' not found
```
The way I made is complicatad but works on allmost every ffmpeg builds. Try to encode a video with an resolution of `1920x1058` without this option.

Reply:
in this specific case we can remove the `fps` option.

Reply:
Checked and it's *not yet* included in the FFmpeg version in RPMFusion. So, it might make sense to wait a bit to introduce that and maybe make it a change for Opencast 9 later.

But I would still apply the suggestion above because that does not change the semantics of your command just the syntax and makes it easier to understand (less unnecessary calculations and no setting defaults)